cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -506,10 +506,15 @@ handleSimpleNode runP p2pMode tracers nc onKernel = do
             (_, Nothing)         -> Map.keys
                                   $ supportedNodeToClientVersions (Proxy @blk)
     when (  ncTestEnableDevelopmentNetworkProtocols nc
```

Comment:
i think there is a type in your commit message: `"Split it into node-to-node and node-to-node"` do you mean node-to-client and node-to-node?